qa/tasks/cephfs/test_admin.py

```diff
@@ -181,9 +181,9 @@ def gen_health_warn_mds_cache_oversized(self, mds_id=None, fs=None):
             raise RuntimeError('Makes no sense to pass both, mds_id as well as '
                                'FS object')
 
-        self.config_set(cs_name, 'mds_cache_memory_limit', '1K')
+        self.config_set(cs_name, 'mds_cache_memory_limit', '50K')
```

Comment:
Why increase it to 50K? With 1K, the chance that the warning is raised by much higher.

Reply:
1K was triggerring the warning on the other mds as well on my local cluster. Hence increased it to 50K